.github/workflows/build-system-test-react.yml

```diff
@@ -0,0 +1,203 @@
+# Description: this workflow runs build mega-apps that have all the connected components.
+
+name: Build System Test Canary - React
+
+permissions:
+  id-token: write # This is required for requesting the JWT
+  contents: read # This is required for actions/checkout
+
+on:
+  schedule:
+    - cron: '*/15 * * * *' # Run every 15 minutes
```

Comment:
It doesn't run every 15 minutes since it's not merged to the default branch (`main`) yet. ([doc](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule))